src/UseCases/Orders/CreateOrderUseCase.java

```diff
@@ -0,0 +1,74 @@
+package UseCases.Orders;
+
+import entities.FoodTruck;
+import entities.Interfaces.*;
+import entities.Regular.RegularOrder;
+import UseCases.DataAccessInterfaces.CustomerRepository;
+import UseCases.DataAccessInterfaces.OrderRepository;
+import UseCases.DataAccessInterfaces.ShopRepository;
+import UseCases.DataAccessInterfaces.VendorRepository;
+import UseCases.OutputBoundary.OrderModel;
+
+import java.util.HashMap;
+
+public class CreateOrderUseCase implements CreateOrderInputBoundary{
+    OrderRepository orderData;
+    CustomerRepository customerData;
+    VendorRepository vendorData;
+    OrderModel orderModel;
+    ShopRepository shopRepository;
+
+    /**
+     * Creates a use case for making order objects
+     *
+     * @param customers the repository containing all customer data
+     * @param vendors the repository containing all vendor data
+     * @param orders the repository containing all order data
+     * @param orderModel the output boundary
+     */
+    public CreateOrderUseCase(CustomerRepository customers, VendorRepository vendors, OrderRepository orders,
+                              OrderModel orderModel, ShopRepository shopRepository){
+        this.orderData = orders;
+        this.customerData = customers;
+        this.vendorData = vendors;
+        this.orderModel = orderModel;
+        this.shopRepository = shopRepository;
+    }
+
+    /**
+     * A method for creating new order objects
+     * @param cart the cart with the items that want to be purchased
+     * @param customer the customer buying the items
+     * @param shopId the food truck selling the items
+     * @return true if the order has been created and false if it hasn't
+     */
+    public boolean createOrder(ICart cart, ICustomer customer, String shopId) {
+        IShop foodTruck = shopRepository.getShop(shopId);
```

Comment:
Why is IShop here?

Reply:
Someone else did orderuse case, this got added to the pull request on accident. It'll get overwritten when that person merges.